internal/storage/postgres/migrations/verifier/20250721200403_delete_policy_constraints.sql

```diff
@@ -0,0 +1,140 @@
+-- +goose Up
```

Comment:
All the triggers and functions which operate solely on the policy table should be a system migration, not a verifier - otherwise each plugin will have to reimplement and copy them